airrohr-firmware/airrohr-firmware.ino

```diff
@@ -528,10 +527,8 @@ String Float2String(const double value) {
 String Float2String(const double value, uint8_t digits) {
 	// Convert a float to String with two decimals.
 	char temp[15];
-	String s;
-
 	dtostrf(value, 13, digits, temp);
-	s = String(temp);
+	String s = temp;
```

Comment:
Don't we need to cast temp? It's a char array assigned to a string now.

Reply:
No worries.

String s = temp; works because there is a String constructor that takes a "const char".

```
class String {
    public:
        // constructors
        String(const char *cstr = "");
        ...
}
```
The tricky bit is that "String s = "bla"" looks like an assignment and one might suspect the assignment operator of String being called, but this is not the case

Reply:
As I said, too many different languages ...